moped-editor/src/views/projects/newProjectView/NewProjectMap.js

```diff
@@ -491,75 +501,74 @@ const NewProjectMap = ({
         />
 
         {/* RENDER LAYERS */}
-        {Object.entries(mapConfig.layerConfigs).map(([sourceName, config]) => {
-          if (isSignalComponent && sourceName !== "drawnByUser") {
-            // hides feature selecting and drawing layers when when component is a signal
-            return null;
-          }
-          if (
-            drawLines === true &&
-            sourceName === "Project_Component_Points_prototype"
-          ) {
-            return null;
-          }
-          if (drawLines === false && sourceName === "CTN") {
-            return null;
-          }
-          return config.layerUrl ? (
-            // If a config has a url, it is needs state to update selected/unselected layers
-            <Source
-              key={config.layerIdName}
-              type="vector"
-              tiles={[config.layerUrl]}
-              maxZoom={config.layerMaxLOD || mapConfig.mapboxDefaultMaxZoom} // maxLOD found in vector tile layer metadata
-            >
-              <Layer
+        {drawLines !== null &&
+          Object.entries(mapConfig.layerConfigs).map(([sourceName, config]) => {
+            if (isSignalComponent && sourceName !== "drawnByUser") {
+              // hides feature selecting and drawing layers when when component is a signal
+              return null;
+            }
+            if (
+              drawLines === true &&
+              sourceName === "Project_Component_Points_prototype"
+            ) {
+              return null;
+            }
+            if (drawLines === false && sourceName === "CTN") {
+              return null;
+            }
+            return config.layerUrl ? (
+              // If a config has a url, it is needs state to update selected/unselected layers
+              <Source
                 key={config.layerIdName}
-                {...createProjectSelectLayerConfig(
-                  featureId,
-                  sourceName,
-                  selectedLayerIds,
-                  visibleLayerIds
-                )}
-              />
-            </Source>
-          ) : (
-            <Source
-              key={config.layerIdName}
-              id={config.layerIdName}
-              type="geojson"
-              data={{
-                ...featureCollection,
-                features: [
-                  ...featureCollection.features,
-                  ...(otherProjectFeaturesCollection?.features ?? []),
-                ].filter(
-                  feature => feature.properties.sourceLayer === sourceName
-                ),
-              }}
-            >
-              <Layer
+                type="vector"
```

Comment:
nothing changed here except wrapping it in ` {drawLines !== null &&` but github did weirdness with highlighting differences